README.md

```diff
@@ -4,42 +4,46 @@ A small library for X window manipulation
 
 ## documentation
 
-Here is the full list of all the function you can use from `libwm`:
-
-    wm_init_xcb();
-    wm_kill_xcb();
-    wm_is_alive(wid);
-    wm_is_ignored(wid);
-    wm_is_listable(wid, mask);
-    wm_is_mapped(wid);
-    wm_get_focus();
-    wm_get_screen();
-    wm_get_windows(wid, **list);
-    wm_get_attribute(wid, attr);
-    wm_get_cursor(mode, wid, *x, *y);
-    wm_set_border(width, color, wid);
-    wm_set_focus(wid);
-    wm_set_cursor(x, y, mode);
-    wm_set_override(wid, mode);
-    wm_teleport(wid, w, h, x, y);
-    wm_move(wid, mode, x, y);
-    wm_remap(wid, mode);
-    wm_resize(wid, mode, w, h);
-    wm_restack(wid, mode);
+Here is the full list of all the functions provided by `libwm`:
+
+```c
```

Comment:
I'm talking about the original markdown specification: http://daringfireball.net/projects/markdown/syntax#precode

So yeah, even if this project is hosted on github *for now*, I prefer not relying on anything specific and keep everything as simple as possible